### PR TITLE
Tag Maxima v0.0.4 [https://github.com/nsmith5/Maxima.jl]

### PR DESCRIPTION
**Pull request by nsmith5**
+ better testing
+ basic windows support
+ many more maxima library functions exported
+ better expression translation

### CODE REVIEW

File: Maxima/versions/0.0.4/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat 0.7.15
```
Comment (tkelman) [inline Maxima/versions/0.0.4/requires]: need 0.9.5 for `take!`

**Comment by tkelman** [comment]
> basic windows support

shouldn't you enable appveyor then rather than deleting the yml file?

**Comment by nsmith5** [comment]
I don't think I really know the windows command line environment well enough to set that up. I can open an issue to try and solicit some help with it perhaps. I've been able to get the package working on a friends window's box, but I'm not in front of one often enough to figure out continuous integration for this package.

**Comment by nsmith5** [comment]
I'll close this PR, ,move the 0.0.4 tag to the updated repo and submit a new PR with the correct compat version.

**Pull request closed without merging**